zulipterminal/model.py

```diff
@@ -284,6 +285,7 @@ def set_narrow(
         pm_with: Optional[str] = None,
         starred: bool = False,
         mentioned: bool = False,
+        active_button: Any = None,
```

Comment:
This parameter isn't being used right now?

Reply:
Yep, sorry, removed this.